api-compact.yaml

```diff
@@ -10,10 +10,10 @@
 #   /enterprise/api/v1/catalogs/
 #   /enterprise/api/v1/catalogs/{id}/
 #   /enterprise/api/v1/catalogs/{id}/courses/
-#   /enterprise/api/v1/enterprise-catalogs/
-#   /enterprise/api/v1/enterprise-catalogs/{uuid}/
-#   /enterprise/api/v1/enterprise-catalogs/{uuid}/course-runs/{course_run_id}/
-#   /enterprise/api/v1/enterprise-catalogs/{uuid}/programs/{program_uuid}/
+#   /enterprise/api/v1/enterprise_catalogs/
+#   /enterprise/api/v1/enterprise_catalogs/{uuid}/
+#   /enterprise/api/v1/enterprise_catalogs/{uuid}/course_runs/{course_run_id}/
+#   /enterprise/api/v1/enterprise_catalogs/{uuid}/programs/{program_uuid}/
```

Comment:
Are we going to revert all this too as per https://github.com/edx/edx-enterprise/pull/202#discussion_r142038744?

Reply:
No, the endpoint paths will use underscores, the URL names will use dashes. This seems to be what DRF defaults to.

Reply:
:+1: